app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class InvitationsController < ApplicationController
+  def new
+    @invitation = Invitation.new
+  end
+
+  def create
+    return unless invitation_params[:invitation_user_id] != current_user.id
+
+    @invitation = Invitation.new(invitation_params)
+    if @invitation.save
+      flash[:success] = 'Successfully sent invitation'
+      redirect_to current_user
+    else
+      flash[:danger] = 'Please select valid input from drop down'
+      render :new
+    end
+  end
+
+  def index
+    @invitation = current_user.event_invitations
+  end
+
+  def accept
+    @attance = EventAttendance.new(event_attendee_id: current_user.id, attendee_event_id: params[:event][:id])
```

Comment:
We have a little bug here, after and invitation is accepted, the accept button still shows and i can still accept the same even again and again.
![Screenshot 2019-09-05 at 12 38 29 PM](https://user-images.githubusercontent.com/10922508/64338599-1db91600-cfda-11e9-895a-58479ad28ce6.png)
![Screenshot 2019-09-05 at 12 38 40 PM](https://user-images.githubusercontent.com/10922508/64338628-2c073200-cfda-11e9-958a-ef9b011e4b33.png)